python/test/block/Test_ACEtk_AngularDistributionBlock.py

```diff
@@ -13,8 +13,8 @@
 from ACEtk import TabulatedAngularDistribution
```

Comment:
Since these PRs are way behind, I'm making the change in the fix/review-1 branch.

Nice catch, I don;t go this far in reviews. Impressive. Very impressive.